src/zkevm_specs/evm/execution/addmod.py

```diff
@@ -0,0 +1,62 @@
+from ..instruction import Instruction, Transition
+from ..opcode import Opcode
+from zkevm_specs.util import FQ, RLC
+
+
+def lt_u256(instruction: Instruction, a: RLC, b: RLC) -> FQ:
```

Comment:
> Could you add a line describing this function? Something like: `Returns 1 when a is lower than b, 0 otherwise`

```suggestion
# Returns 1 when a is lower than b, 0 otherwise
def lt_u256(instruction: Instruction, a: RLC, b: RLC) -> FQ:
```